docs/api_specs/status_events_specs.md

```diff
@@ -37,6 +37,7 @@ Indicates that an invalid exit is occurring. It should be challenged.
 ```
 
```

Comment:
the document is saying that `invalid_ife_challenge` is not implemented yet, is this a blocker for mainnet? cc @Pongch

Reply:
These are stale comments, everything that here said "not implemented" is implemented. Nice catch!